src/proc/hdr-merge.cpp

```diff
@@ -23,11 +24,22 @@ namespace librealsense
         if (!depth_frame)
             return false;
 
-        if (!depth_frame.supports_frame_metadata(RS2_FRAME_METADATA_SUBPRESET_SEQUENCE_SIZE))
-            return false;
-        if (!depth_frame.supports_frame_metadata(RS2_FRAME_METADATA_SUBPRESET_SEQUENCE_ID))
+        if (!depth_frame.supports_frame_metadata(RS2_FRAME_METADATA_SEQUENCE_SIZE) ||
+            !depth_frame.supports_frame_metadata(RS2_FRAME_METADATA_SEQUENCE_ID))
+        {
+            ++_frames_without_requested_metadata_counter;
+            // warning message wiill be sent to user if more than NUMBER_OF_FRAMES_WITHOUT_METADATA_FOR_WARNING
+            // frames are received without the needed metadata params
+            if (_frames_without_requested_metadata_counter > NUMBER_OF_FRAMES_WITHOUT_METADATA_FOR_WARNING)
+            {
+                LOG_WARNING("HDR Merge filter cannot process frames because relevant metadata params are missing");
+                _frames_without_requested_metadata_counter = 0;
```

Comment:
Make it a one-shot message by resetting the counter only when stream restarts.
In addition only advance it if it hasn't reached NUMBER_OF_FRAMES_WITHOUT_METADATA_FOR_WARNING yet

Reply:
done